### PR TITLE
KeepingYouAwake Support

### PR DESCRIPTION
**Pull request by zharmany**
Added support for [KeepingYouAwake](https://github.com/newmarcel/KeepingYouAwake), an open-source [Caffeine](https://itunes.apple.com/us/app/caffeine/id411246225?mt=12) clone.

### CODE REVIEW

File: mackup/applications/keepingyouawake.cfg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[application]
+name = KeepingYouAwake
+
+[configuration_files]
+Library/Preferences/info.marcel-dierkes.KeepingYouAwake.plist
```
Comment (lra) [inline mackup/applications/keepingyouawake.cfg]: Can you end this line with a newline character?

Comment (zharmany) [inline mackup/applications/keepingyouawake.cfg]: Hi, Laurent,

Sure! Just made the change.

Do I need to issue another pull request? This was actually my first on GitHub!

Thanks for your work on this project,
-Zac

Comment (lra) [inline mackup/applications/keepingyouawake.cfg]: nope, all good
